src/Sdk/DTWebApi/WebApi/TaskHttpClient.cs

```diff
@@ -127,6 +127,22 @@ public sealed class TaskHttpClient : TaskHttpClientBase
                                       cancellationToken);
         }
 
+        public Task<ActionDownloadInfoCollection> ResolveActionDownloadInfoAsync(
```

Comment:
do we need this? since it didn't do anything special other than calling the generated httpclient.